cypress/integration/metaSpec.js

```diff
@@ -0,0 +1,27 @@
+describe('The example page with imgix meta tags', () => {
+  it('successfully loads', () => {
```

Comment:
I'm not really sure what this is testing - it might be better to include a more descriptive test name.

If I'm understanding the test correctly, it seems like it's just testing the behaviour of Cypress itself, which I don't think is necessary to test? If so this test might not be necessary?

Reply:
This is a good point, I meant to test that when the page has a meta tag imgix.js was aware of its existence, ie `util.getMetaTagValue(key)` does not return `undefined`. As written this test doesn't do that. Will fix.